src/widget/form/groupinviteform.cpp

```diff
@@ -149,8 +153,29 @@ void GroupInviteForm::onGroupInviteRejected()
     int index = inviteLayout->indexOf(groupWidget);
     groupInvites.removeAt(index);
 
-    groupWidget->deleteLater();
-    inviteLayout->removeWidget(groupWidget);
+    deleteInviteButtons(groupWidget);
+}
+
+void GroupInviteForm::deleteInviteButtons(QWidget* widget)
+{
+    QList<QPushButton*> buttons = widget->findChildren<QPushButton*>();
+
+    if (acceptButtons.contains(buttons.at(0)))
```

Comment:
I still think it is better to replace this construct with something like:
```
acceptButtons.subtract(&QSet::fromList(buttons));
rejectButtons.subtract(&QSet::fromList(buttons));
```
(just an example, I don't know if it is correct)
See http://doc.qt.io/qt-5/qset.html#subtract and http://doc.qt.io/qt-5/qset.html#fromList for details.


As it's much cleaner and verbose.